test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueListenerTests.cs

```diff
@@ -86,99 +83,65 @@ public QueueListenerTests(TestFixture fixture)
             messageFromCloud = await queue.GetMessageAsync();
             Assert.Equal(2, messageFromCloud.DequeueCount);
 
-            processor.Original = messageFromCloud;
-
             await listener.ProcessMessageAsync(new StorageQueueMessage(messageFromCloud), TimeSpan.FromMinutes(10), CancellationToken.None);
-        }
 
-        private class TestQueueProcessor : QueueProcessor
-        {
-            private CloudQueue _queue;
-            private CloudQueue _poisonQueue;
-            private CloudQueueMessage _original;
+            // Make sure the message was processed and deleted.
```

Comment:
Won't this be the case w/o your changes? I thought the issue only happened when users force upgraded storage versions?